draft-ietf-rats-architecture.md

```diff
@@ -378,78 +372,58 @@ Attesting Environments are designed specifically with claims collection in mind.
-It is important to note that the new Attesting Environment cannot alter
-the content about its own measurements. If the Attesting Environment
-would be able to do that, Layered Attestation would become unfeasible.
-
-In this example the duty of being the Attesting Environment is now
-taken over by the (U)EFI / BIOS / Firmware that was the Attested
-Environment before. This transfer of duty is the essential part of
-Layered Attestation. The (U)EFI / BIOS / Firmware now is the Attesting Environment.
-The next Target Environment is, in this example, a bootloader. There are
-potentially multiple kernels to boot, the decision is up to the bootloader.
-Only a bootloader with intact integrity will make an appropriate decision. Therefore, Claims about
-the integrity of a bootloader are now collected by the freshly appointed Attesting Environment
-that is the (U)EFI / BIOS / Firmware. Collected Claims have to be stored by the current
-Attesting Environment in a similar shielded and secured manner, so that the next Attesting Environment
-is not capable of altering the collection of claims stored.
-
-Continuing with this example, the bootloader is now in charge of collecting Claims
-about the next execution environment. The next execution environment in this example
-is the kernel to be booted up. Analogously, the next transfer of duties in this
-Layered Attestation example occurs: The duty of being an Attesting Environment is
-transferred to a successfully measured kernel. In this sequence, the kernel is now collecting
-additional Claims and is storing them in a secure and shielded manner.
-
-~~~
-    [Henk: we might have to define what successful
-    means in this example and beyond]
-~~~
-
-The essence of this example is a cascade of staged boot environments. Each
-environment (after the initial one that is a root-of-trust) has the duty
-of measuring its next environment before it is started. Therefore, creating
-a layered boot sequence and correspondingly enabling Layered Attestation.
+Attesting Environment A, the read-only BIOS in this example,
+has to ensure the integrity of the bootloader (Target Environment B).
+There are
+potentially multiple kernels to boot, and the decision is up to the bootloader.
+Only a bootloader with intact integrity will make an appropriate decision. Therefore, these Claims have to be measured securely.
+At this stage of the boot-cycle of the
+device, the Claims collected typically cannot be composed into Evidence.
+
+After the boot sequence is started, the BIOS conducts the
+most important and defining feature of layered attestation, which is that
+the successfully measured Target Environment B
+now becomes (or contains) an Attesting Environment for the next layer.
+This procedure in Layered Attestation is sometimes called "staging".
+It is important to note that the new Attesting Environment B cannot alter
+any Claims about its own Target Environment B, since those Claims were signed
```

Comment:
When we say "signed" here are we implying a signing operation done by A in the cryptographic sense, i.e. using A's private key?  If so, I'm not sure this is the necessary precondition.  Measurements might just be "sealed" in the RoT at a certain boot stage for later use by the lead attester.

Reply:
Good point, updated.